### DISCUSSION TITLE
New YouTube Button for Media

### DISCUSSION THREAD

**Discussion by peppersgc** [Purfview/IMDb-Scout-Mod#242] [Ideas (implemented)]
Can we get a new YouTube search button for the actual movies and TV shows, separate from, or replacing, the button for a trailer on YouTube that's already right in front of us on the IMDb page. That just occurred to me, that that button is very antiquated. It was there when I first started using IMDb Scout, LONG before Purfview took over with the very welcome mod. Right now, we have to manually remove the quotation marks, and the word "trailer", from the search box to find the actual movie or TV show. It would also be extremely helpful if "&sp=EgIYAg%253D%253D" were added after the search term. It simply filters to the top of the search results the videos with an over 20 minute duration.

Full Example:

'name': 'YouTube Movies & TV',
      'icon': 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAMAAABEpIrGAAAAP1BMVEUAAAD/AAD/AAD/AAD/AAD/AAD/AAD/AAD/AAD/AAD/AAD/////8PD/0ND/wMD/kJD/gID/UFD/QED/ICD/EBBwI9TdAAAACnRSTlMAwECAMBDwsGAgE90AyAAAAHFJREFUOMvVkkkOgCAMRYXKYBEH9P5nVdCoIdImuuJt/1t0+E09KGkBxAMAK9UVtwJfEe2ZayygD8NgEZMEJIh5RwlxUIk3cy7ITHDDwgiun2hhxwdKSIzrP8GHD0MSa7KHYk/NPYt9N1sYrnJcaathA2rBFn/0Nk1kAAAAAElFTkSuQmCC',
      'searchUrl': 'https://www.youtube.com/results?search_query=%search_string%+%year%&sp=EgIYAg%253D%253D'},

Thank you in advance for the consideration.

**Comment by Purfview** [comment]
Looks like for TV shows the best search is not to specify anything:

`https://www.youtube.com/results?search_query=The+Handmaid's+Tale`

If you add any filter then it can't be found:

`https://www.youtube.com/results?search_query=The+Handmaid's+Tale&sp=EgIYAg%253D%253D`

**Comment by peppersgc** [comment]
I see what you're saying, but that's intentional. We don't look for shows on YouTube to pay for them, and adding the filter removes the "Pay to Watch" options streamed by Google. Also, A Handmaids Tale isn't a good test, because it's too actively monitored for removal. For example:

https://www.youtube.com/results?search_query=The+Golden+Girls

And filtered.

https://www.youtube.com/results?search_query=The+Golden+Girls&sp=EgIYAg%253D%253D

**Reply by Purfview** [reply]
Added "Youtube Filtered" in v23.5
